src/main/routes.ts

```diff
@@ -128,6 +128,7 @@ export class Routes {
     app.get(CONTACT_US, errorHandler(new ContactUsGetController().get));
     app.get(`${APPLICANT_TASK_LIST_URL}/:caseId`, errorHandler(new GetCaseController().fetchAndRedirectToTasklist));
     app.get(`${RESPOND_TO_APPLICATION}/:caseId`, errorHandler(new GetCaseController().fetchAndRedirectToTasklist));
+    app.get('/applicant/task-list', (req, res) => res.redirect('/task-list/applicant'));
```

Comment:
@hmcts-abduldaim This will redirect all applicants to new tasklist page irrespective of caseType right ?